### DISCUSSION TITLE
Exercice calcul Surface

### DISCUSSION THREAD

**Discussion by PBYetml** [PBYetml/PROG_SLO1_24_25#32] [Announcements]
info pour : 
@Kazanaris @REXI38 @Renato1226 @VolutedPenny235 @fredericgoyet @HENRIIIIIIIII @Nintalbdly @Alex84628 @Z1nghy @P60B40 @Hayatache 

Contrainte : 

les variables longueur, largeur ou hauteur, rayon doivent être déclarées en entier !!! 

CDC minimal : dans la fonction main, uniquement instruction permettant de faire des calculs (sans affiche, sans demande de utilisateur) 

CDC avancé : toujours dans la fonction main reprendre le principe minimal, ajout des fcts permettant de faire des demandes utilisateurs (affichage + saisie clavier) 

CDC expert : reprendre le CDC avancé mais réaliser un fichier headerFile + un autre fichier source permettant d'avoir 4 fcts différentes

**Comment by HENRIIIIIIIII** [comment]
oui merci

**Comment by PBYetml** [comment]
suivi de projet - voir issue #33
